src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -63,6 +64,11 @@
 
 public class AllChecksTest extends AbstractModuleTestSupport {
 
+    private static final Set<String> INTERNAL_MODULES = Collections.unmodifiableSet(
+            new HashSet<>(Collections.singletonList(
```

Comment:
same here

Reply:
Since I moved `INTERNAL_MODULES` to `Definitions` class, it needs unmodifiable object.